### PR TITLE
Add debug to invoke tasks

### PR DESCRIPTION
**Pull request by JonZeolla**
# Contributor Comments

Add a debug flag to invoke tasks

This depends on https://github.com/SeisoLLC/easy_infra/pull/90

## Pull Request Checklist

Thank you for submitting a contribution to `easy_infra`!

In order to streamline the review of your contribution we ask that you review and comply with the below requirements:

- [x] Rebase your branch against the latest commit of the target branch
- [x] If you are adding a dependency, please explain how it was chosen
- [x] If manual testing is needed in order to validate the changes, provide a testing plan and the expected results
- [x] If there is an issue associated with your Pull Request, align your PR
  title with [this documentation](https://help.github.com/en/articles/closing-issues-using-keywords)

### CODE REVIEW

File: tasks.py [hunk 1]
```diff
@@ -155,18 +165,32 @@ def build(c):  # pylint: disable=unused-argument
...
-            path=str(CWD), target=target, rm=True, tag=first_image, buildargs=buildargs
-        )[0]
+        try:
+            image = CLIENT.images.build(
+                path=str(CWD), target=target, rm=True, tag=first_image, buildargs=buildargs
+            )[0]
+        except docker.errors.BuildError as build_err:
+            LOG.exception("Failed to build target %s, retrieving and logging the more detailed build error...", target)
+            iterator = iter(build_err.build_log)
+            while not finished:
```
Comment (JonZeolla) [inline tasks.py:175]: This needs a `finished = False` before here, I may not have pushed my commit...

File: tasks.py [hunk 2]
```diff
@@ -155,18 +165,32 @@ def build(c):  # pylint: disable=unused-argument
...
+                try:
+                    item = next(iterator)
+                    LOG.error("%s", item)
+                except StopIteration:
+                    finished = True
+
```
Comment (JonZeolla) [inline tasks.py:181]: Also forgot the `sys.exit(1)` at the bottom here

File: tasks.py (hunk 1 above)
Comment (JonZeolla) [reply tasks.py:175]: Ah I pushed it to the wrong branch.  Will remedy this morning

**Review by seanctech — APPROVED** [review]
LGTM!

**Pull request merged**